tests/Backend/Mixed/SharingToSpecificProjectsTest.php

```diff
@@ -86,6 +86,15 @@ public function testShareBucketToProject($backend)
             $project = reset($sharedBucket['sharingParameters']['projects']);
             $this->assertEquals($targetProject['id'], $project['id']);
             $this->assertEquals($targetProject['name'], $project['name']);
+
+            $this->assertNotEmpty($this->_client->getSharedBucketDetail($targetProject['id'], $bucketId));
```

Comment:
Nepokrývá tohle část toho co testuje testShareBucketDetailVisibility ? Tím, jak jsou ty testy drahé (hlavně setup/teardown) tak pokud se to otestuje tady, tak by se to nemuselo testovat jinde.